homework/SeleniumBasics2/EpamSiteNavigateTests.cs

```diff
@@ -0,0 +1,82 @@
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Interactions;
+using System.Collections.Generic;
+
+namespace SeleniumBasics2
+{
+    [TestFixture]
+    public class EpamSiteNavigateTests
+    {
+        private IWebDriver _driver;
+        private const string link = "https://www.epam.com/";
+        
+        [SetUp]
+        public void InicializeBrowser()
+        {
+            _driver = new ChromeDriver();
+            _driver.Manage().Window.Maximize();
+            _driver.Navigate().GoToUrl(link);
+        }
+
+        [Test]
+        public void CheckTheListOfCountriesInTheCareerButtonTest()
+        {
+            _driver.FindElement(By.XPath("//*[@href = '/careers']")).Click();
+            var careerOffice = _driver.FindElement(By.XPath("//*[@class='tabs__ul js-tabs-links-list']"));
+            var careerElements = careerOffice.FindElements(By.XPath(".//div"));
+
+            Assert.That(careerElements.Count(), Is.EqualTo(3), "Invalid number of countries!");
+        }
+
+        [Test]
+        public void CheckForAWordInTheSearchBoxTest()
+        {
+            var linkAutomation = "https://www.epam.com/search?q=Automation";
+
+            _driver.FindElement(By.XPath("//*[@class = 'header-search__button header__icon']")).Click();
+            _driver.FindElement(By.XPath("//*[@id='new_form_search']")).SendKeys("Automation");
+            _driver.FindElement(By.XPath("//*[@class = 'header-search__submit']")).Click();
+            _driver.FindElement(By.XPath("//*[@class = 'search-results__items']"));
+            _driver.FindElement(By.XPath(".//*[@class = 'search-results__item']")).Equals(5);
```

Comment:
is test pass? There are 10 items on page. There is no asser, you just have useless boolean variable